source/terms/information_elements/mids_information_elements_draft.csv

```diff
@@ -1,29 +1,29 @@
-mids:,CollectingNumber,Collecting Number,An identifier given to the specimen at the time it was recorded. Often serves as a link between field notes and a specimen record.,For many biological collections the collector in combination with collectingNumber has been used as an identifier for the specimen in literature. It also helps in the identification and validation of the collecting location.,It is recommended to use “s.n.” (sine numero - without number) or equivalent if not present,"BPL-1001, 154/23, 254, s.n",http://www.w3.org/2000/01/rdf-schema#Class,Yes (Biology),No,original,2024/05/16,
-mids:,Media,Media,A list (concatenated and separated) of media associated with the specimen.,To allow a user to locate mediafiles of the specimen,It is strong recommendation to follow Audiovisual Core standards; identifier for IIIF manifest and/or image,https://www.unimus.no/felles/bilder/web_hent_bilde.php?id=14501286&type=jpeg,http://www.w3.org/2000/01/rdf-schema#Class,Yes (Biology),Yes,original,2024/05/16,
-mids:,GeologicalAge,Geological Age,"Refers to the geological age of a Earth Science specimen (i.e. Fossil, Rock, Mineral or Meteorite) and can be any kind of stratigraphic age, isotopically determined age or structural age.","To provide information about the age of the geological, mineralogical, fossil or meteorite specimen","If relevant, it is recommended follow the International Commission on Stratigraphy chart, and that litho- and biostratigraphy, where available, should also be provided.",,http://www.w3.org/2000/01/rdf-schema#Class,Yes (Geology & Palaeontology),Yes,original,2024/05/16,
-mids:,TypeStatus,Type Status,"An indication of the nomenclatural type status of the specimen, where a null value is taken to mean ""Assumed not to be a type"".","The nomenclatural type specimens are some of the most important specimens in a collection, particularly for taxonomic research. They anchor the taxon name to a physical object.",It is strongly recommended to follow the list of terms included in the relevant codes of nomenclature.,"Holotype, Isotype, Syntype, Cotype, Epitype, Neotype, Lectotype",http://www.w3.org/2000/01/rdf-schema#Class,Yes (Biology),No,original,2024/05/16,
-mids:,InstitutionID,Institution ID,An identifier for the institution having custody of the object(s),To allow a user (human or machine) to locate the physical location of the specimen.,"It is strongly recommended to use a globally unique persistent and resolvable identifier such as GrSciColl, ROR.","https://ror.org/01xtthb56, https://www.gbif.org/grscicoll/collection/772f9e37-4643-452b-82b4-a06550283096, https://www.gbif.org/grscicoll/institution/e3d4dcc4-81e2-444c-8a5c-41d1044b5381",http://www.w3.org/2000/01/rdf-schema#Class,Yes (all),Yes,original,2024/05/16,
+“s.d.” (sine diem - without date) or equivalent if not present or not known.",2023-12-08 | s.d.,http://www.w3.org/2000/01/rdf-schema#Class,Yes (Biology),No,original,2024/05/16,

```

Comment:
Add some more examples of allowed date formats similar to https://dwc.tdwg.org/list/#dwc_eventDate?